src/utils/validation.ts

```diff
@@ -104,7 +104,7 @@ function validateElement(
     };
     validations?: ResolvedCustomValidation[];
   },
-  repeat: number
+  repeat: any
```

Comment:
number | undefined seemed good to me

Reply:
Typescript complains.  If I go with that, then I need to add the undefined guard back in.